### DISCUSSION TITLE
Avalanche Community Proposal Meeting 05-03-2024

### DISCUSSION THREAD

**Discussion by JacobEverly** [avalanche-foundation/ACPs#80] [Announcements]
# Avalanche Community Proposal Meeting 05-03-2024

**Meeting Date/Time:** 05-03-2024, 16:00 UTC  
**Meeting Duration:** 67 minutes
**Moderator:** Jacob Everly

## Purpose of Avalanche Community Proposal Meetings

A meeting for discussing and coordinating initiatives that affect the Avalanche Protocol. The main purpose of this protocol is for the organization of meetings and discussions around proposed Avalanche Community Proposals (ACP).

## Who is invited?

Any protocol developer, ACP author, researcher, or any other individual that would be impacted by the proposed changes are invited to attend the meetings.

## Links

- **Sign Up Form:** [Google Form](https://forms.gle/kM92NTp35Yg93s5v5)
- **Youtube Link:** [Youtube Link](https://www.youtube.com/watch?v=cxLyT90j2hs)

## Preparation Material

To prepare for this meeting please thoroughly review the following materials so we are all on the same page when we start the meeting:

- **ACP-68 Discussion:** [GitHub Discussion](https://github.com/avalanche-foundation/ACPs/discussions/68)
- **ACP-77 Post:** [GitHub ACP-77](https://github.com/avalanche-foundation/ACPs/blob/main/ACPs/77-reinventing-subnets.md)
- **ACP-77 Discussion:** [GitHub Discussion](https://github.com/avalanche-foundation/ACPs/discussions/78)

## Agenda

1. **Discuss ACP-77:**
   - Confusion around the proposal
   - Concerns people have with the proposal
   - Feature requests for the proposal

## Introduction

The meeting focused on discussing ACP 77 concerning the new subnet creation flow and permission validation in the Avalanche network.

## Overview of ACP 77

- **Dhruba Basu:** Presented an overview, explaining that the core thesis of ACP 77 is to maintain subnets as separate from the primary network while still retaining underlying guarantees. The proposal aims to simplify the creation of subnets, reduce costs, and increase flexibility for subnet validators.

### Separation of Subnet Validators and Primary Network Validators

- Current challenges include a high entry barrier for new subnets, requiring a deposit of 2,000 AVAX to validate the primary network.
- ACP 77 proposes removing this requirement, allowing for greater autonomy and simplified management for subnets.

### Staking and Rewards Flexibility

- Subnets would have more control over their validator sets and could set their staking requirements and reward mechanisms independently from the primary network.

### Pchain Activity and Continuous Fee Mechanism

- The removal of the AVAX deposit is compensated by a new fee mechanism on the Pchain to support increased activity and security.

### Concerns and Alternative Proposals

- **Aaron Buchwald:** Raised questions about who should bear the cost of fees for subnet validators and how the Pchain should handle evictions. He suggested alternatives where subnets themselves could be responsible for these fees, improving user experience and reducing complexities.

## Community Feedback

Various members expressed support for the flexibility offered by ACP 77 but also highlighted concerns about potential complexities in managing validator fees and the need for clear migration strategies for existing subnets.

## Decision Points

There was a consensus on the need for flexibility in managing subnets and a general agreement that ACP 77 could significantly streamline the process. However, detailed mechanisms for implementation and migration still need to be ironed out based on community feedback and technical feasibility.

## Next Steps

- Community members are encouraged to provide further input and feedback on the ACP 77 discussion thread on GitHub.
- Avalanche Labs will continue to refine the proposal, addressing community concerns and feedback to reach a final implementable version.

## Closing Remarks

The meeting concluded with a reminder that the proposal is a collaborative effort and all community input is valuable in shaping the future of the Avalanche network.

**Action Items:**
- **Community Members:** Continue to provide feedback on the GitHub discussion thread.
-  Update ACP 77 based on feedback and prepare for potential implementation or revision.

**Comment by notexeditor** [comment]
My ideal #APower subnet should:

■ Pay *cheapest* possible $AVAX rent for P-chain use,
□ with each validator paying their individual rent.
□ Validators should bond $APOW native tokens *only*;
□ failure to fulfill validator duties (incl. rent) should cause slashing of rewards.

Since $APOW inflation is already happening on the C-chain, I'd like to reward my #APower validators with:

■ *part* of the $APOW TX fees, where low subnet TPS would imply a high fraction of fees as rewards, and a high TPS a low fraction;

■ and AWM/Teleporter *bridge tolls*, where the toll tax should be *progressive*.
